src/panels/CreateFleetPanel.ts

```diff
@@ -1,4 +1,132 @@
 import { ContainerServiceFleetClient, Fleet } from "@azure/arm-containerservicefleet";
+import { BasePanel, PanelDataProvider } from "./BasePanel";
+import { Uri, window } from "vscode";
+import { MessageHandler, MessageSink } from "../webview-contract/messaging";
+import {
+    InitialState,
+    ProgressEventType,
+    ToVsCodeMsgDef,
+    ToWebViewMsgDef,
+    ResourceGroup as WebViewResourceGroup,
+} from "../webview-contract/webviewDefinitions/createFleet";
+import { TelemetryDefinition } from "../webview-contract/webviewTypes";
+import { ResourceGroup as ARMResourceGroup, ResourceManagementClient } from "@azure/arm-resources";
+import { ReadyAzureSessionProvider } from "../auth/types";
+import { getAksFleetClient, getResourceManagementClient } from "../commands/utils/arm";
+import { getResourceGroups } from "../commands/utils/resourceGroups";
+import { failed } from "../commands/utils/errorable";
+
+export class CreateFleetPanel extends BasePanel<"createFleet"> {
```

Comment:
can/should this be a const string variable? it appears a number of times throughout the file